sktime/forecasting/base/_base.py

```diff
@@ -380,7 +380,7 @@ def fit(self, y, X=None, fh=None):
         self._update_y_X(y_inner, X_inner)
 
         # check forecasting horizon and coerce to ForecastingHorizon object
-        fh = self._check_fh(fh)
```

Comment:
`_check_fh` is not a pure/static method, it is already using parameters from `self`. So instead of passing here, I would suggest to simply get `self._cutoff` at the start, `_cutoff = self._cutoff`, and add it in the docstring on the list of accessed attributes if not already present there. That way, you need to change much less of the lines.

Reply:
done